### PR TITLE
mapdiffbot test

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

Type here what the purpose of your PR is.
Mention if you tested your changes. If you changed a map, make sure you used the mapmerge tool.

Mention if your PR is related to an issue.
You can type "Fixes issue #6" and issue 6 will be automatically closed when your pull request is merged. This is much easier for everyone.

Edit the changelog at the bottom for changes that are noticeable by the players. Remove it if this isn't the case.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.
Include [admin] in the title if it's something admin related.

#### Changelog

:cl:  
rscadd: Added new things  
rscdel: Removed old things  
bugfix: fixed a few things  
wip: This is work in progress  
tweak: tweaked a few things  
soundadd: added a new sound thingy  
sounddel: removed an old sound thingy  
imageadd: added some icons and images  
imagedel: deleted some icons and images  
spellcheck: fixed a few typos  
experimental: This is experimental  
/:cl:

### CODE REVIEW

**Comment by Cyberboss** [comment]
reopen this and push a commit

**Review by ThatLing — APPROVED** [review]

**Comment by Cyberboss** [comment]
Working Directory: `minimap --disable hide-space "S:\Website\Cyberbot\App_Data\Requests\MapDiffs\Repositories\yogstation13\yogstation\_maps/map_files/Yogstation/yogstation.2.1.3.dmm.old_map_diff_bot`

That explains it...

**Comment by Cyberboss** [comment]
push another commit

**Comment by Cyberboss** [comment]
fuck, another pls

File: config/admins.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # NOTE: if the rank-name cannot be found in admin_ranks.txt, they will not be adminned! ~Carn #
 # NOTE: syntax was changed to allow hyphenation of ranknames, since spaces are stripped.      #
 ###############################################################################################
-Kn0ss0s	=	Head Coder
+Nichlas0010	=	Head Coder
```
Comment (ThatLing) [inline config/admins.txt]: That's not how you write our lord, AsV9's name.

**Review by ThatLing — DISMISSED** [review]

**Comment by Amelia0010** [comment]
rip @mapdiffbot

**Comment by Cyberboss** [comment]
Sorry, i've been really busy push another commit please

**Comment by Cyberboss** [comment]
Ok, now its running, it doesn't like one of the strings your putting out. https://dextraspace.net/Cyberbot/Operations/yogstation13/yogstation/3124/error_log.txt

@spaceManiac , did you fix this yet?

**Comment by SpaceManiac** [comment]
Pushed some updates to SpacemanDMM which when combined with #3137 will allow yogstation to parse.

**Comment by Cyberboss** [comment]
bot updated, give her another commit

**Comment by SpaceManiac** [comment]
`thread 'main' panicked at 'attempt to multiply with overflow', src\tools\dmm.rs:291:28`

`dmm.rs:291: curr_key = 52 * curr_key + base_52_reverse(ch).unwrap();`

Map has big keys like `"ZxW"` and [BYOND has trouble with keys](https://secure.byond.com/forum/?post=2340796#comment23770802) higher than `"ymo"`, so I don't support them either. Run `convert.py` from [mapmerge2](https://github.com/tgstation/tgstation/tree/master/tools/mapmerge2) on your map to automatically fix this. I was running without overflow checks which is why I didn't catch this.

**Comment by Cyberboss** [comment]
This should be working now btw if you feel like pushing another commit

**Pull request closed without merging**